src/presets/cloudflare-pages.ts

```diff
@@ -138,7 +138,7 @@ function comparePaths(a: string, b: string) {
 
```

Comment:
@pi0 I can remove changes in this file from this PR, just wanted your opinion

Reply:
Thanks looks good 👍🏼 as always seperate PRs are better for changelog btw.